gym/wrappers/record_episode_statistics.py

```diff
@@ -46,6 +124,10 @@ def __init__(self, env: gym.Env, deque_size: int = 100):
         self.return_queue = deque(maxlen=deque_size)
         self.length_queue = deque(maxlen=deque_size)
         self.is_vector_env = getattr(env, "is_vector_env", False)
+        if self.is_vector_env:
```

Comment:
Is this compatible when the `VectorListInfo` is applied to the vector envs?

Reply:
You mean a scenario like this 
`RecordEpisodeStatistics(VectorListInfo(gym.vector.make('CartPole-v1', num_envs=3)))`
 instead of this
`VectorListInfo(RecordEpisodeStatistics(gym.vector.make('CartPole-v1', num_envs=3)))`
?
This won't work, I've stated in docstring of `VectorListInfo` that it need to be the outermost wrapper if there are other wrappers performing operations on info. Other than that, if there is a way (which atm I'm not aware of) of checking all the wrappers chaining I could raise a warning or an exception if `VectorListInfo` is not the outermost

Reply:
Yes, we could add a test that checks the `env` (and all sub wrappers) are not instances of `VectorListInfo`. There is similar code in the `tests/wrappers/utils.py`. 
Hopefully, this will stop an issue in the future